readme.md

```diff
@@ -77,6 +84,20 @@ A sample configuration json:
 For application registration and client id and client secret, please refer to:
 [Blue Button 2.0 API Docs - Try the API](https://bluebutton.cms.gov/developers/#try-the-api)
 
+FHIR requests retry parameters:
+
+retrySettings (Optional), parameters for exponential back off retry algorithm
+
+| retry parameter | value (default)      | Comments                         |
+| --------------- | -------------------- | -------------------------------- |
+| initInterval    | 5000                 | back off interval in milli sec   |
+| maxAttempts     | 3                    | max retry attempts               |
+| retryableCodes  | [500, 502, 503, 504] | error response codes to retry on |
+
```

Comment:
Information about how to "turn-off" retries might be good to add here. 

For example:

To completely disable retry behavior, set  `retryableCodes` to an empty LIST.